assets/css/_docs.scss

```diff
@@ -160,7 +160,7 @@ html.docs {
     input {
       border: none;
       border-bottom: 2px solid $defaultLiteColor;
-      font-size: 1.3rem;
+      font-size: 16px;
```

Comment:
The browser disagrees

![screencast](https://cloud.githubusercontent.com/assets/1402241/18595376/48435300-7bf9-11e6-841a-25db95b26d1d.gif)